src/fields/custom-field.js

```diff
@@ -1,19 +1,19 @@
 import React, { PropTypes as PT, cloneElement } from 'react';
 import { Field } from 'redux-form';
-import { createInlineError, fieldClasses } from './field-utils';
+import { fieldClasses } from './field-utils';
```

Comment:
Not used in custom field. It is used in the labeled field component though. But the labeled field component handles everything for the user. While the custom component lets the users handle everything themselves.